docs/sources/helm-charts/mimir-distributed/configure/configure-redis-cache.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configure Redis cache"
+menuTitle: "Redis cache"
+description: "Learn how to configure Grafana Mimir to use external Redis as cache"
+---
+
+# Configure Redis cache
+
+Beside built-in support for Memcached, Mimir also supports Redis for the chunks-cache, index-cache, results-cache and metadata-cache. To use Redis, disable the built-in Memcached
+from `mimir-distributed` helm chart by making sure the cache configuration is not enabled. To disable Memcached either set `*-cache.enabled` to false as shown in the following yaml or remove the whole `*-cache`
+block from values file so that default setting disables Memcached. Removing the whole block is more recommended.
```

Comment:
You suggested good things. However,  I adjust thing a bit because this guide is also to be used by people who are installing Mimir for the first time, not necessarily the ones who has had Mimir installed with Memcached beforehand. Therefore the point "disable the built-in Memcached instances deployed by the `mimir-distributed` Helm chart" might not be really accurate. I will commit something like this below, but let me know if you have more things to add.

```suggestion
Besides support for Memcached, Mimir also supports Redis for the chunks-cache, index-cache, results-cache and metadata-cache. To use Redis, deploy Redis instances, disable the built-in Memcached configuration flag in  values.yaml of `mimir-distributed` Helm chart, and then configure Mimir to use Redis.

To disable Memcached, remove any `chunks-cache`, `index-cache`, `metadata-cache` and `results-cache` configuration from your Helm `values.yaml` file. Alternatively, explicitly disable each of the Memcached instances by setting `enabled` to `false`:
```